pkg/config/config_test.go

```diff
@@ -105,6 +105,7 @@ func TestNewConfigBasedOnExistingWithNewConfigOptions(t *testing.T) {
 		WithTokens(DataSetTokens{WriteLog: "writeLogOpt1"}),
 		WithBufferSettings(*bufCfg),
 		WithServerHostSettings(*hostCfg),
+		WithDebug(true),
```

Comment:
Do we have any test with debug true which verify that `session_key` is indeed present?

Reply:
Yes. It's here - `TestNewEventWithMeta` - https://github.com/scalyr/dataset-go/pull/62/files#diff-59675dba27959ea0941fcebce68b40688dbd7e6dcf545f9009659ba5a6133633R141